runtime/README.md

```diff
@@ -281,6 +281,35 @@ Command Code: `0x5349_4756` ("SIGV")
 | chksum      | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
 | fips\_status | u32      | Indicates if the command is FIPS approved or an error.
 
+### LMS\_SIGNATURE\_VERIFY
+
+Verifies an LMS signature. The hash to be verified is taken from
+Caliptra's SHA384 accelerator peripheral.
+The supported parameter set is limited to: sha256/192, w=4, and h=15 (the same used for the Caliptra image)
+
+Command Code: `0x4C4D_5356` ("LMSV")
+
+*Table: `LMS_SIGNATURE_VERIFY` input arguments*
+
+| **Name**              | **Type** | **Description**
+| --------              | -------- | ---------------
+| chksum                | u32      | Checksum over other input arguments, computed by the caller. Little endian.
+| pub\_key\_tree\_type  | u8[4]    | LMS public key algorithm type. Must equal 12.
+| pub\_key\_ots\_type   | u8[4]    | LM-OTS algorithm type. Must equal 7.
+| pub\_key\_id          | u8[16]   | Private key identifier
+| pub\_key\_digest      | u8[24]   | Public key hash value
```

Comment:
Same as for the signature, should the public key be passed as a single buffer? The public key format is spec-defined: https://datatracker.ietf.org/doc/html/rfc8554#section-4.3

Reply:
Although to be fair, we do split apart signatures and public keys for ECDSA into x/y and r/s.

Reply:
See response above, this was mainly done this way for consistency. But it is a lot of params. I am okay with flattening to just the pub_key and signature params if wanted.

Reply:
Ack, ok to leave as-is